### PR TITLE
MIR-689 marc xml

### PR DESCRIPTION
**Pull request by yagee-de**

### CODE REVIEW

File: mir-module/src/main/resources/xsl/mods2marcxml.xsl [hunk 1]
```diff
@@ -0,0 +1,2820 @@
...
+			<xsl:with-param name="tag">
+			<xsl:choose>
+				<!-- changes attribute 'restrictionOnAccess' to 'embargo'-->
+				<xsl:when test="@type='embargo'">506</xsl:when>
+				<xsl:when test="@type='useAndReproduction'">540</xsl:when>
+			</xsl:choose>
+			</xsl:with-param>
+			<xsl:with-param name="subfields">
+				<marc:subfield code='a'>
+					<xsl:value-o
```
Comment (kkrebs) [inline mir-module/src/main/resources/xsl/mods2marcxml.xsl]: this should be configurable using a mycore.property

File: mir-module/src/main/resources/xsl/mods2marcxml.xsl [hunk 2]
```diff
@@ -0,0 +1,2820 @@
...
+				<xsl:value-of select="$nstring"/>
+			</xsl:otherwise>
+		</xsl:choose>
+	</xsl:template>
+
+	<xsl:template match="mods:mods">
+		<marc:record>
+			<marc:leader>
+				<!-- 00-04 -->
+				<xsl:text>     </xsl:text>
```
Comment (kkrebs) [inline mir-module/src/main/resources/xsl/mods2marcxml.xsl]: these 4 whitespaces seems to be ignored in XML output
<marc:leader>nam  22     uu 4500</marc:leader>

File: mir-module/src/main/resources/xsl/mods2marcxml.xsl [hunk 3]
```diff
@@ -0,0 +1,2820 @@
...
+			<xsl:if test="mods:genre[@authority='marc']='remote sensing image'">
+				<marc:controlfield tag="007">ar||||||</marc:controlfield>
+			</xsl:if>
+			<xsl:if test="mods:genre[@authority='marc']='map'">
+				<marc:controlfield tag="007">aj||||||</marc:controlfield>
+			</xsl:if>
+			<xsl:if test="mods:genre[@authority='marc']='globe'">
+				<marc:controlfield tag="007">d|||||</marc:controlfield>
+			</xsl:if>
+			<marc:controlfield tag="008">
```
Comment (kkrebs) [inline mir-module/src/main/resources/xsl/mods2marcxml.xsl]: controlfields are typical generated from bib system. perhaps we should delete them?

File: mir-module/src/main/resources/xsl/mods2marcxml.xsl [hunk 4]
```diff
@@ -0,0 +1,2820 @@
...
+				<xsl:variable name="typeOf008"><xsl:apply-templates mode="ctrl008" select="mods:typeOfResource"/></xsl:variable>
+				<!-- 00-05 -->
+				<xsl:choose>
+					<xsl:when test="mods:recordInfo/mods:recordContentSource[@authority='marcorg']">
+						<xsl:value-of select="mods:recordInfo/mods:recordCreationDate[@encoding='marc']"/>
+					</xsl:when>
+					<xsl:otherwise>
+						<xsl:text>      </xsl:text>
```
Comment (kkrebs) [inline mir-module/src/main/resources/xsl/mods2marcxml.xsl]: these leading whitespaces are also ignored
`<marc:controlfield tag="008">s           ||||||||||||||||||||||</marc:controlfield>`

File: mir-module/src/main/resources/xsl/mods2marcxml.xsl (hunk 1 above)
Comment (kkrebs) [inline mir-module/src/main/resources/xsl/mods2marcxml.xsl]: I'm not shure if this should be hard coded ...

File: mir-module/src/main/resources/xsl/mods2marcxml.xsl (hunk 2 above)
Comment (yagee-de) [reply mir-module/src/main/resources/xsl/mods2marcxml.xsl]: works for me, please see source of the output not the browser output: https://dbttest.thulb.uni-jena.de/dbt-master/receive/dbt_mods_00032675?XSL.Transformer=marcxml

File: mir-module/src/main/resources/xsl/mods2marcxml.xsl (hunk 4 above)
Comment (yagee-de) [reply mir-module/src/main/resources/xsl/mods2marcxml.xsl]: pleas see source of file. looks ok to me

**Comment by yagee-de** [comment]
requires #264

**Pull request merged**